vectortile/src/main/scala/geotrellis/vectortile/internal/package.scala

```diff
@@ -121,9 +121,12 @@ package object internal {
     *
     */
   private[vectortile] def toProjection(point: (Int, Int), topLeft: Point, resolution: Double): Point = {
+    val translatedX: Double = topLeft.x + (resolution / 2)
+    val translatedY: Double = topLeft.y + (resolution / -2)
```

Comment:
Changing the translation to a full point (from half a point here) further reduces the delta between expected and actual results. It's still ~4 meters off after that though.

Reply:
(This presumes that the input coordinate is correct, though (see below).)